### ISSUE TITLE
Unknown option `atomic'

### ISSUE THREAD

**Issue by wickedest** [lerna/lerna#2400]
## Expected Behavior

Braking change.  Expect that `lerna publish` to work as before.

## Current Behavior
`lerna publish` fails with:
```
lerna notice cli v3.20.1
lerna info versioning independent
lerna info ci enabled
lerna info require-scripts enabled
...
lerna info execute Skipping releases
lerna info git Pushing tags...
lerna ERR! Error: Command failed: git push --follow-tags --no-verify --atomic origin master
lerna ERR! error: unknown option `atomic'
```

## Possible Solution
Related to #2393

## Context
This breaks our CI.  We are blocked and cannot publish our modules.

## Your Environment

| Executable | Version |
| ---: | :--- |
| `lerna --version` | 3.20.1 |
| `npm --version`  | 6.1.0 |
| `node --version` | 8.11.3 |

**Comment by evocateur** [comment]
I guess https://github.com/lerna/lerna/commit/2b9b210c0b6ac69853ffb01f0dbac9109ab419c5 wasn't sufficient. What version of `git` are you using?

**Comment by wickedest** [comment]
git version 2.1.4

**Comment by wickedest** [comment]
While 2.1.4 is laughably old (Dec 17 2014), it is part of a cypress docker base image that uses node:8.12.  I had a bit of a dig through the lerna documentation, and I couldn't see anything about a minimum version of git.  It might be good to publish that.

**Comment by evocateur** [comment]
Yeah, I spent the last 20 minutes or so cursing google's inability to find what version of git is supported by various CI providers. I think 2.1.4 is the latest provided by debian 8 (jessie)'s core packages.

In any event, my fallback was too strict, I'll be releasing a patch once CI passes

**Comment by evocateur** [comment]
Published in [v3.20.2](https://github.com/lerna/lerna/releases/tag/v3.20.2)
